pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -39,6 +38,7 @@ var folderPermissionTranslation = map[models.PermissionType][]string{
 		dashboards.ActionFoldersRead,
 	}...),
 	models.PERMISSION_EDIT: append(dashboardPermissionTranslation[models.PERMISSION_EDIT], []string{
+		ac.ActionDashboardsCreate,
```

Comment:
question: this would affect the previous migration, right? Is it fine to change it like that?

Reply:
So this permission was already included for folder permission edit, but was added to dashboard by mistake (with dashboard scope so it does not work). We could drop all `dashboard:create` permissions with dashboard scopes